distribution/ddf-common/src/main/resources/etc/shell.init.script

```diff
@@ -32,27 +32,38 @@ log:list = { log:get ALL } ;
 service:get = { $.context getService ($.context getServiceReference $args) };
 
 env = { shell:set $args }
-more = { shell:less $args }
+more = { shell:less -F $args }
 
-prompt = "[1m${USER}[0m@${APPLICATION}(${SUBSHELL})> "
+\#prompt = { "[1m${USER}[0m@${APPLICATION}(${SUBSHELL})> "?};
```

Comment:
[KARAF-5031](https://issues.apache.org/jira/browse/KARAF-5031) Subshell doesn't show in prompt